homework2.ts

```diff
@@ -0,0 +1,64 @@
+interface AreaInterface {
```

Comment:
The name for interface should be ShapeInterface, and in shape, we have methods for calculating area and get name of shape.